### PR TITLE
39- styling the page hero- Delnia

### PR DESCRIPTION
**Pull request by DelniaAlipour**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

- Create Hero.js component-Imported to Home.js
- Created Hero.css file

Addresses #39 
Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Comment by DelniaAlipour** [comment]
Hi @mferryRV , Here is the screenshot of Hero section:
hero

File: client/src/App.js
Comment (mferryRV) [inline client/src/App.js]: This just looks like a formatting change?

File: client/src/components/Hero.css
Comment (mferryRV) [inline client/src/components/Hero.css:6]: This is fine, but generally when using `em` and a 16px font, we choose multiples of 0.125 (2px), 0.25 (4px), 0.5 (8px). These will resolve to whole px numbers. This may not result in "pixel perfect" replications of the designs, but it is more readable and more visually consistent.

No changes needed here but this is an informational comment. PX values should be whole numbers, EM should be certain fractions that result in whole PX numbers.

File: client/src/components/Hero.css
Comment (mferryRV) [inline client/src/components/Hero.css]: Like this should probably be 3em rather than 3.06em. The difference is 1px and visually minimal.

Good design systems will follow standard sizes.

File: client/src/components/Hero.js
Comment (mferryRV) [inline client/src/components/Hero.js]: I'd prefer clearer separation between code and content. Something like:
```js
const content = {
 h1: "Black Atlantic Space-Time",
 }
 
 return <div>
     <h1>{content.h1}</h1>
   </div>
 ```
 
The current version still mixes text and JSX, which I'm not a fan of.

File: client/src/pages/Home.js
Comment (mferryRV) [inline client/src/pages/Home.js]: These formatting changes are likely to cause conflicts with @christina-mifsud 's  major overhaul of `Home.js`

Please coordinate on release - I think this PR should be merged after #53

**Review by mferryRV — COMMENTED** [review]
Please take any JSX out of the `content` object so all the values are simple strings, then this is ready to approve.

I would recommend waiting until after #53 to merge.

File: client/src/App.js
Comment (DelniaAlipour) [reply client/src/App.js]: Yes, this happens automatically after I uninstall and reinstall Prettier.

File: client/src/components/Hero.css
Comment (DelniaAlipour) [reply client/src/components/Hero.css:6]: Thanks for clarification.

**Review by mferryRV — APPROVED** [review]
All good - currently there's no `<Hero />` component on the home page. I trust us to add it later if needed.

**Pull request merged**
